cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
```

Comment:
Delegates that have 2 strong references on each other, 1 of them should be marked as weak
suggest -> weak var delegate: PaymentViewControllerDelegate?